consult.el

```diff
@@ -927,22 +928,26 @@ CAND is the currently selected candidate."
       (let ((start 0)
             (end (length cand)))
         ;; Find match end position, remove characters from line end until matching fails
-        (let ((step 16))
-          (while (> step 0)
-            (while (and (> (- end step) 0)
-                        (completion-all-completions input (list (substring cand 0 (- end step))) nil 0))
-              (setq end (- end step)))
-            (setq step (/ step 2))))
+        (unless (eq consult-line-point-placement 'start-of-line)
```

Comment:
The end computation must be done in every case, since end is used later.

Reply:
I currently do the end computation for `start-of-match` and for `end-of-match`. That should be enough, no? The third case, `start-of-line` doesn't need it.

Reply:
Right, I misread. Sorry.

Reply:
Now that the start-of-line case is handled earlier, I removed this check, but I still think it was correct before. 😉